python/GangaDirac/Lib/Backends/DiracBase.py

```diff
@@ -95,6 +96,8 @@ class DiracBase(IBackend):
         'settings': SimpleItem(defvalue={'CPUTime': 2 * 86400},
                                doc='Settings for DIRAC job (e.g. CPUTime, BannedSites, etc.)'),
         'credential_requirements': ComponentItem('CredentialRequirement', defvalue=DiracProxy),
+        'blockSubmit' : SimpleItem(defvalue=True, 
+                               doc='Shall we use the experimental block submission?'),
```

Comment:
To call it experimental and then implement it as the default seems odd. I would just delete "experimental".